cloudman/projman/migrations/0001_initial.py

```diff
@@ -1,13 +1,16 @@
-# Generated by Django 2.0 on 2019-06-21 18:05
+# Generated by Django 2.2.6 on 2020-02-05 11:57
```

Comment:
Should update Django to >=2.2.9 to avoid the known security vulnerability.

Reply:
I regenerated the migrations with latest but the version didn't affect the migrations.